docs/low-code-spark/gems/source-target/file/delta.md

```diff
@@ -530,7 +530,7 @@ object writeDeltaSCD2 {
 
 ### SCD3
 
-Using the same customer tables as in our merge example above, output and configurations for SCD3 merge will look like below.
+Using the same customer tables as in our merge example above, output and configurations for SCD3 merge will look like below. Let's track change for `customer_zip_code` by adding a column to show the previous value.
```

Comment:
why must we add a column? shouldn't Delta be capturing this history in the log? @slyons

Reply:
Maybe the delta log is what enables us to capture these previous values?
<img width="632" alt="image" src="https://user-images.githubusercontent.com/20563764/183525881-b8278ce4-f61e-40da-b9b7-a770d8c39aa0.png">